packages/core/core/src/Config.js

```diff
@@ -1,224 +1,117 @@
 // @flow
```

Comment:
@wbinnssmith Since this is a public API, should it go in `src/public`? Also haven't thought through if we need to have a public and an internal version.

Reply:
I think we should probably adhere to this more, and this would be a good first step :)